src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs

```diff
@@ -8,13 +8,14 @@ public static class EnvironmentNames
         public const string Development = "DEVELOPMENT";
         public const string AT = "AT";
         public const string TEST = "TEST";
+        public const string TEST2 = "TEST2";
         public const string DEMO = "DEMO";
         public const string PREPROD = "PREPROD";
         public const string PROD = "PROD";
 
         public static string[] GetTestEnvironmentNames() => new []{ Development, AT, TEST, DEMO };
         public static string GetTestEnvironmentNamesCommaDelimited() => string.Join(",", Development, AT, TEST, DEMO);
-        public static string GetEnvironmentNamesCommaDelimitedForGMT() => string.Join(",", DEMO, PREPROD, PROD);
+        public static string GetEnvironmentNamesCommaDelimitedForGTM() => string.Join(",", DEMO, PREPROD, PROD);
```

Comment:
GetTestEnvironmentNamesCommaDelimited & GetNonProdEnvironmentNamesCommaDelimited need to include TEST2. Maybe more?